docs/installation/installation.md

```diff
@@ -1,5 +1,8 @@
 # Installation
 
+> **Note**
```

Comment:
This `> **Note**` formatting looks a bit weird to me in the markdown preview.

Reply:
The icon and blue color seems to be a github thing.... locally I see this:

<img width="790" alt="Screen Shot 2022-10-03 at 8 24 27 PM" src="https://user-images.githubusercontent.com/20706237/193708843-54fa36c7-5622-4ce2-a244-7df142f2b89e.png">

 Is that what looks weird to you?  If so I can try to change it to look more like my local shot above